pkg/driver/node/mounter/pod_mounter.go

```diff
@@ -36,24 +36,28 @@ const targetDirPerm = fs.FileMode(0755)
 // This is mainly exposed for testing, in production platform-native function (`mountSyscallDefault`) will be used.
 type mountSyscall func(target string, args mountpoint.Args) (fd int, err error)
 
+type fileGroupID func(path string) (gid uint32, err error)
```

Comment:
Seems like this is only exposed for testing, and I feel like it would be cleaner if we just change the `gid` of the parent directory in our tests to test different cases via [os.Chown](https://pkg.go.dev/os#Chown).

Reply:
Using `os.Chown` requires root privileges (or `CAP_CHOWN` capability) and adding root privileges to unit tests is not a best practice.

Reply:
Agh, makes sense to keep it as is then!